### DISCUSSION TITLE
about 4G LTE ip camera app

### DISCUSSION THREAD

**Discussion by RosaiK** [quasarframework/quasar#13477] [Ideas / Proposals]
Hello , all guys

I want to make a solar-powered video surveillance camera,
4G LTE sent to the cloud,
Needless to say, the client must use the QUASAR,
Ask the server, which platform should be better.
And how can customers who are distributed in many regions only see their own camera content.

Please give me some pointers when you have time, thank you very much.

**Comment by suleimanas** [comment]
Hi, sorry i didn't get much from your question. let's break it down :) and correct me if i'm wrong

1. "I want to make a solar-powered video surveillance camera, 4G LTE sent to the cloud," that's up to you to use any power source you want, but if you mean that you are building an app that record or stream media to the cloud. use WebRTC it's ready now.

2. "Needless to say, the client must use the QUASAR,": it's up to you 

3. "Ask the server, which platform should be better." need more info, who ask the server. or are you asking which server to use?

4. "And how can customers who are distributed in many regions only see their own camera content."

NEED MORE INFO :)

**Reply by RosaiK** [reply]
Very grateful for your help. Sorry for my late reply! ! !

webrtc is a module built into the browser. Really good way to call.

My original intention was to use a 4g camera, infrared detection of human movement, then turn on the camera, and transmit the video to the cloud server (I don't know what kind of cloud service program can handle video storage, query, and forwarding. Similar to firebase, but video data).
The client APP can receive, alarm, and view playback, or live broadcast.

However, because the video data is too large and the 4G data consumption is too much, most products now store the video in the camera's memory card. When necessary, the data in the camera can be called via the internet line.

Thanks! ! !

**Reply by suleimanas** [reply]
Yes you can do it. look for a camera with good api but why send it to the cloud it will be expensive.

it would be cheaper to send the videos to your own device or the user's device, or any hosting with good bandwidth then stream it on demand to the user again using webrtc

**Reply by RosaiK** [reply]
thank you for your reply.
The installation site is a farm and there is no wifi network.

**Reply by suleimanas** [reply]
No one said anything about wifi, just saying it can be done. and anyway it has nothing to do with quasar
Good luck.

**Reply by RosaiK** [reply]
Than you very much!
Good luck!

**Reply by RosaiK** [reply]
Thank you for your kind help.
But I am very strange, I am considering js, vue, quasar to build client app,
This is also IDEA's forum, why it has been emphasized that my problem has nothing to do with quasar.

Thank you, your comments are very useful. I wish you all the best!

**Reply by suleimanas** [reply]
You are missing the point my friend, it's not IDEA's forum it's (Ideas / Proposals about Quasar) not howto and it's not stackoverflow.
And if you considering using js, vue, quasar to build client app as you said, it'll be good for you, it's great choice after all

**Reply by RosaiK** [reply]
Thanks for your help, looks like I posted in the wrong place, sorry!
Good luck!

**Comment by MilosPaunovic** [comment]
All that sounds like an interesting idea, but how exactly does it relate to Quasar?

**Reply by RosaiK** [reply]
The client application uses the quasar framework.
